.github/workflows/FormatCheck.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         # TODO: Change the call below to
         #       format(".")
         run: |
-          julia  -e 'using Pkg; Pkg.add(PackageSpec(name = "JuliaFormatter", version="1.0.45"))'
+          julia  -e 'using Pkg; Pkg.add(PackageSpec(name = "JuliaFormatter", version="1.0.60"))'
```

Comment:
> Maybe it would make sense to permanently fix the formatter version, so that development is not delayed by new versions. But then add a GitHub workflow to periodically check if a new version is available and perhaps automatically create a PR like the compat workflow.

_Originally posted by @efaulhaber in https://github.com/trixi-framework/Trixi.jl/issues/1842#issuecomment-1941490640_

Would be cool to have a workflow like this, but I don't have the ~time~ motivation to do this.

Reply:
I see.